### PR TITLE
Add a link to help with Forking a repo

### PR DESCRIPTION
**Pull request by jarvisluong**
I also fixed some spelling mistakes and some Vietnamese writing styles.

### CODE REVIEW

File: translations/README-vn.md [hunk 1]
```diff
@@ -216,7 +219,7 @@ Ok, bây giờ bạn có thể bắt đầu:
 ## Về nguồn video
 
 Một vài video chỉ xem được khi bạn tham gia vào các lớp học online trên Coursera, EdX, hay Lynda.com. Các lớp đó được gọi là MOOC.
-Đôi khi các lớp không có sẵn, và bạn phải đợi một vài tháng đến khi chúng được mở lại, do đó bạn không thể truy cập vào video được.
+Đôi khi các lớp chư mở, và bạn phải đợi một vài tháng đến khi chúng được mở lại, do đó bạn không thể truy cập vào video được.
```
Comment (dangkhoasdc) [inline translations/README-vn.md]: chưa mở :D

**Comment by letientai299** [comment]
Thanks @dangkhoasdc for your help on reviewing.

**Comment by jarvisluong** [comment]
@dangkhoasdc @letientai299 I have fixed the typo :)

**Comment by letientai299** [comment]
@jwasham We have done the review. Please merge this PR.

**Comment by jwasham** [comment]
Thanks!

**Pull request merged**